Sources/AvatarView.swift

```diff
@@ -73,30 +145,42 @@ open class AvatarView: UIView {
         contentMode = .scaleAspectFill
         layer.masksToBounds = true
         clipsToBounds = true
-        addSubview(imageView)
         imageView.contentMode = .scaleAspectFill
         imageView.frame = frame
+        addSubview(imageView)
         imageView.image = avatar.image ?? getImageFrom(initals: avatar.initals)
-        setCorner(radius: nil)
+        setCorner(radius: self.radius)
     }
     
     // MARK: - Open setters
     
     open func set(avatar: Avatar) {
+        self.avatar = avatar
         imageView.image = avatar.image ?? getImageFrom(initals: avatar.initals)
     }
     
+    open func set(font: UIFont) {
+        self.font = font
+    }
+    
+    open func create() {
+        prepareView()
```

Comment:
This should be done in the init

Reply:
It used to be like that, but in that case whenever user set a font, he will have to recreate the images again. Another solution might be, have another `init` which sets everything from the initialization.

Reply:
I mean in `init`, user already create an avatar in any case. Once she change the font, she will recreate it again. Thats why I created another function for creating avatar.

Reply:
> It used to be like that, but in that case whenever user set a font, he will have to recreate the images again. Another solution might be, have another init which sets everything from the initialization.

I think this is what the `convenience init` would be good for, so lets add another `init` that initializes everything.